release-notes/10.0/preview/preview1/csharp.md

```diff
@@ -2,13 +2,37 @@
 
 .NET 10 Preview 1 includes the following C# features & enhancements:
 
-- [Feature](#feature)
+- [`nameof` in unbound generics](#unbound-generic-support-for-nameof)
+- [Implicit span conversions](#implicit-span-conversions)
+- [`field` backed properties](#field-backed-properties)
+- [Modifiers on simple lambda parameters](#modifiers-on-simple-lambda-parameters)
+- [Preview feature - String literals in data section](#preview-feature-string-literals-in-data-section)
 
 C# 14 updates:

```

Comment:
Is this the header other workloads are using? Maybe:

```suggestion
For more about C# 14 updates:

```

Reply:
This is consistent with other release notes. I'll leave this and let @jamesmontemagno decide.